src/pe_reports/data/__init__.py

```diff
@@ -0,0 +1 @@
+"""Initialize database folder for tests to pass."""
```

Comment:
I don't understand this comment. Are you saying that the tests weren't passing unless that comment was present?

Reply:
Updated to """The pe-reports data directory."""  [here](https://github.com/cisagov/pe-reports/pull/114/commits/6e727fe0b430b8bd02de8f3e5ae7f27c778f1925) Just a bad comment. The __init__ is so we can treat the directory as a package.

*think the "tests" we were referring to was the flask pre-commit check

Reply:
Got it, thanks!

BTW, the [current linting error](https://github.com/cisagov/pe-reports/runs/4927222432?check_suite_focus=true) should get resolved once you pull in the latest code from the base branch.